Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -295,6 +295,11 @@ MediaPlayerPrivateGStreamer::~MediaPlayerPrivateGStreamer()
         gst_bus_set_sync_handler(bus.get(), nullptr, nullptr, nullptr);
         g_signal_handlers_disconnect_matched(m_pipeline.get(), G_SIGNAL_MATCH_DATA, 0, 0, nullptr, nullptr, this);
     }
+
+    GST_DEBUG("ODH_REPORT_AVPIPELINE_STATE_DESTROY");
```

Comment:
please test if creation and removal of empty video tag is triggering this report

Reply:
Checked,no reports because MediaPlayerPrivateGstreamer object is not created in that case, if we set empty string to src on existing video object then MediaPlayerPrivateGstreamer is destroyed and new one is not created.